pkg/build/controller/policy/policy.go

```diff
@@ -11,6 +11,7 @@ import (
 
 	buildapi "github.com/openshift/origin/pkg/build/api"
 	buildclient "github.com/openshift/origin/pkg/build/client"
+	buildlister "github.com/openshift/origin/pkg/build/generated/listers/build/internalversion"
```

Comment:
@smarterclayton we should not move to using caches for the build policy stuff without considering all the implications. Here we really want to make sure that there's not another build in Running state before running another build (in the case of the serial policy). 

/cc @bparees